docs/UserGuide.md

```diff
@@ -60,38 +60,55 @@ Oops! There seems to be an error, please check the format of your command again.
 
 Adds a fosterer to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS animal/ANIMAL_NAME t/AVAILABILITY t/HOUSING_TYPE t/TYPE_OF_ANIMAL_FOSTERED/WANTED…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS housing/HOUSING_TYPE availability/AVAILABILITY animal/ANIMAL_NAME animalType/TYPE_OF_ANIMAL_FOSTERED/WANTED t/TAG…`
 
 Parameters:
 * `NAME`: Name of fosterer
 * `PHONE_NUMBER`: Phone number of fosterer
 * `EMAIL`: Email of fosterer
 * `ADDRESS`: Address of foster family
-* `ANIMAL_NAME`: Name of animal fostered (if fosterer is currently fostering an animal / “NotAvailable”)
-* `AVAILABILITY`: Available / NotAvailable
-* `HOUSING_TYPE`: HDB / Condo / Landed
-* `TYPE_OF_ANIMAL_FOSTERED` (if not available): current.Dog / current.Cat
-* `TYPE_OF_ANIMAL_WANTED` (if available): able.Dog / able.Cat
+* `HOUSING_TYPE` (case-sensitive) : HDB / Condo / Landed / nil
+* `AVAILABILITY` (case-sensitive) : Available / NotAvailable / nil
+* `ANIMAL_NAME`
+  * `ANIMAL_NAME (availability: NotAvailable)`: Name of animal fostered
+  * `ANIMAL_NAME (availability: nil)`: nil
+  * `ANIMAL_NAME (availability: Available)`: nil
```

Comment:
I think maybe you can clarify the notation you are using at the top!!!
like the (field : value) notation you are using, so that readers know this means if the availability field == NotAvailable

Reply:
Changed!